.github/composer-config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "config": {
+        "preferred-install": {
+            "symfony/form": "source",
```

Comment:
what is the reason that those 3 components will be installed as source, and all other as dist?
what makes those 3 special?

Reply:
ohh I guess the answer ist in the title of this PR:
https://github.com/symfony/symfony/pull/33637